### PR TITLE
Added inner circle color

### PR DESCRIPTION
**Pull request by Maryom**
Regarding #16

### CODE REVIEW

File: Source/AIFlatSwitch.swift [hunk 1]
```diff
@@ -15,6 +15,11 @@ import UIKit
     
     // MARK: - Public
     
+    /**
+     circleLayer, is the layer which appears inside the circle.
+     */
+    @IBInspectable open var circleLayer = CAShapeLayer()
```
Comment (cocoatoucher) [inline Source/AIFlatSwitch.swift:21]: Is there any chance you can rename this to something like backgroundLayer and have an @IBInspectable backgroundLayerColor?

**Review by cocoatoucher — CHANGES_REQUESTED** [review]

**Comment by cocoatoucher** [comment]
Hi Mariam,
Thanks very much, it looks great. I added a comment as a feedback, just some small naming change request.
However. is it possible for you to make the changes indicated in the "Contribution Guidelines" in the Readme of the project?
Then, I'll merge this and release the new pod version

File: Source/AIFlatSwitch.swift (hunk 1 above)
Comment (Maryom) [reply Source/AIFlatSwitch.swift:21]: Like this?

```
@IBInspectable open var backgroundLayer = CAShapeLayer()
    
/**
     Color for the inner circle.
*/
@IBInspectable open var backgroundLayerColor: UIColor = UIColor.clear
```

Comment (cocoatoucher) [reply Source/AIFlatSwitch.swift:21]: Yes exactly and you need to implement didSet{} of backgroundLayerColor to react to changes on the go

**Pull request merged**